2200060/1.py

```diff
@@ -0,0 +1,9 @@
+for t in range(1,int(input())+1):
+    N, K = map(int,input().split())
+    A = list(map(int,input().split()))
+    a = []
+    for i in range(1,N+1):
+        if i not in A:
+            a.append(i)
+    b = tuple(a)
+    print(f'#{t}',*b)
```

Comment:
깔끔한 풀이네요! 👍
```suggestion
    print(f'#{t}',*a)
```
tuple로 변환하지 않으셔도 괜찮아요! 😄